docker/node/server.js

```diff
@@ -462,32 +462,21 @@ function doorStuck(socket) {
 	socket.emit('doorStuck');
 }
 function playNext() {
-	var Old = SERVER.ACTIVE;
-	if (SERVER.ACTIVE.volat) {
-		var elem = SERVER.PLAYLIST.first;
-		for (var i = 0; i < SERVER.PLAYLIST.length; i++) {
-			if (elem == SERVER.ACTIVE) {
-				SERVER.ACTIVE = SERVER.ACTIVE.next;
-				delVideo({ index: i });
-				break;
-			}
-			elem = elem.next;
-		}
-	} else {
-		// Check for a volatile color tag
-		if ("colorTagVolat" in Old.meta) {
-			var elem = SERVER.PLAYLIST.first;
-			for (var i = 0; i < SERVER.PLAYLIST.length; i++) {
-				if (elem == SERVER.ACTIVE) {
-					setVideoColorTag(i, false, false);
-					break;
-				}
-				elem = elem.next;
-			}
-		}
-		SERVER.ACTIVE = SERVER.ACTIVE.next;
+	const active = {
```

Comment:
I don't think extra variable actually makes the code any cleaner. Any occurrences of `active.node` might as well be `SERVER.ACTIVE`, and `active.position` is only used once so can be inlined.

Reply:
`active.position` is also utilised in `delVideo()` where it's required as the position is also sent to the clients.

Reply:
Ah, so it is. Nevermind!